packages/ui/atomic/health-check/tsconfig.json

```diff
@@ -6,8 +6,8 @@
     "module": "Node16",
     "target": "ESNext",
     "moduleResolution": "Node16",
-    "sourceMap": false,
-    "declaration": false,
+    "sourceMap": true,
+    "declaration": true,
```

Comment:
Turn on declaration files to run the E2E tests. But don't include in the build that will be published